### ISSUE TITLE
Content principles: guidance on how to chunk content

### ISSUE THREAD

**Issue by m-sullivan7** [cagov/design-system#159]
The seventh content principle, [Organize content strategically](https://docs.google.com/document/d/1CLZqTK-Q2_1Gte6HGVrfW53F4eRTFkWGU5y5-n9Hohw/edit?usp=sharing), includes this standard:

> Keep it [content] in digestible chunks.

with this reasoning

> 
Dense content can discourage people from even attempting to read it. When you break up information into chunks, people can more easily find and understand it.

and this guidance:

> Break up large blocks of content into smaller chunks.
> Small paragraphs are easier to read. They also signal when a new topic begins.
> Empty space between paragraphs gives readers little breaks that make the content easier to read.

In writing this guidance, the content team acknowledges that chunking content requires a certain amount of skill and practice. Since our principles are intended for people new to content design, we want to explore whether additional guidance would be useful for readers. In particular:

- How to chunk content
- Where chunking falls in ordering content by importance
- Creating examples that take poorly chunked content and show how it can be better presented
- Finding another term for _chunk_

This is a v2 content principle task.

**Comment by m-sullivan7** [comment]
@thepegisin and @LizNobriga do we still want to do this work? OK if we do, just thought I'd ask now that the Design System is launched.

**Comment by LizNobriga** [comment]
I think we do. At a minimum, we could look for a good reference link to include.

**Comment by m-sullivan7** [comment]
Then let's do this in the next sprint for the Design System. I'll look for some good candidates for articles. 🙂

**Comment by m-sullivan7** [comment]
I did some initial research through internet searching. A lot of chunking guides are in the education field, often discussing how to divide videos.

I found 6 that were related to content/UX design. Most of them fell short of the mark and didn't feel worthy of us giving an implicit seal of approval to. The best one I found was from @thepegisin's favorite [Nielsen Norman Group](https://www.nngroup.com/articles/chunking/). They also created a [video on the topic](https://www.nngroup.com/videos/chunking/), but it's about why chunking is valuable rather than how to do it.
